src/Visualization/Python/GenerateXdmf.py

```diff
@@ -247,6 +247,10 @@ def parse_args():
 
 
 if __name__ == "__main__":
+    if (sys.version_info < (3, 0)):
+        print("WARNING: you are attempting to run this script with python 2, "
```

Comment:
- I think this warning should be in the `generate_xdmf` function, since it’s not specific to when it’s called as “main”.
- Please use Python’s `logging` module (we shouldn’t even start adding `print`s to our Python code). Just `import logging` at the top and use `logging.warning` instead of `print`. Then in the `if __name__ == “__main__”` clause, you can add the following line to default-configure logging (we can add a CLI option for the log level later if we want):

    ```python
    logging.basicConfig(level=logging.INFO)
    ```